SwiftGen.playground/Pages/Storyboard-Demo.xcplaygroundpage/Contents.swift

```diff
@@ -86,4 +101,12 @@ validateVC.title
    not known by the storyboard. But it should work correctly in a real project. */
 // let cgu = UIStoryboard.Name.Wizzard.createAccountViewController
 
+let segue = UIStoryboard.Segue.Message(rawValue: "CustomSegue")!
```

Comment:
Given that your string associated with the `.CustomSegue` case above is `"Custom"` not `"CustomSegue"` I guess this doesn't actually work? Probably forgot to update your example then? (reading that from GitHub so not actually tested here)